usaspending_api/search/v2/elasticsearch_helper.py

```diff
@@ -91,8 +91,9 @@ def get_total_results(keyword, index_name):
     try:
         response = CLIENT.search(index=index_name, body=query)
         return response['hits']['total']
-    except Exception as es1:
-        return es1
+    except Exception:
+        logging.exception("There was an error connecting to the ElasticSearch instance.")
```

Comment:
will the ES client raise descriptive error messages? If yes, it would be good to log the exception msg for tracking down problems on deployed instances